src/main/java/uk/gov/ons/census/action/model/dto/FulfilmentRequestDTO.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ons.census.action.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class FulfilmentRequestDTO {
```

Comment:
Think this might need to include an `individualCaseId` field as well, as in the [case-processor](https://github.com/ONSdigital/census-rm-case-processor/blob/master/src/main/java/uk/gov/ons/census/casesvc/model/dto/FulfilmentRequestDTO.java#L18). 

There's information in the event dictionary around the `individualCaseId` field with regards to PQs, but I'm not certain if it's in scope for this work?

Reply:
@liamtoozer Individual cases aren't in our scope at all yet, I'd rather have ignore them until we know what the requirements are.